inc/Engine/Media/AboveTheFold/AJAX/Controller.php

```diff
@@ -166,7 +166,22 @@ private function create_object( $image, $keys ) {
 				// For other types, add the first non-empty key to the object.
 				foreach ( $keys as $key ) {
 					if ( isset( $image->$key ) && ! empty( $image->$key ) ) {
-						$object->$key = $this->sanitize_image_url( $image->$key );
+						if ( is_array( $image->$key ) ) {
```

Comment:
I know that it's not my call here but I'll add a small comment here:
why not keeping the code as is and only case the $image->$key to be array? this will handle this case with less code, what do u think?